apps/vs-code-react/src/state/store.ts

```diff
@@ -2,10 +2,12 @@
 import { dataMapSlice } from './DataMapSlice';
 import { designerSlice } from './DesignerSlice';
 import { workflowSlice } from './WorkflowSlice';
+import { projectSlice } from './projectSlice';
 import { configureStore } from '@reduxjs/toolkit';
 
 export const store = configureStore({
   reducer: {
+    project: projectSlice.reducer,
     workflow: workflowSlice.reducer,
```

Comment:
@ccastrotrejo Would this 'workflow' make sense for the reducer name for containing overview, review, export?

Reply:
Yeah, I really don't have anything in mind to change that name to.